<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphAnalysisDockWidgetBase</class>
 <widget class="QDockWidget" name="GraphAnalysisDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph layer manager</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="selectionButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../../../Downloads/QgsGraphLayerPlugin/graphlayer.qrc">
          <normaloff>:/graphlayerselect.png</normaloff>:/graphlayerselect.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Nodes</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="tbl_NodeAttributes">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Attribute</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Edges</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableWidget" name="tbl_EdgeAttributes">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Attribute</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Simple edition tools</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addNodeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../../../Downloads/QgsGraphLayerPlugin/graphlayer.qrc">
          <normaloff>:/graphlayeraddnodes.png</normaloff>:/graphlayeraddnodes.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addEdgeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../../../Downloads/QgsGraphLayerPlugin/graphlayer.qrc">
          <normaloff>:/graphlayeraddedges.png</normaloff>:/graphlayeraddedges.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeNodeButton">
        <property name="icon">
         <iconset resource="../../../../../../../../../Downloads/QgsGraphLayerPlugin/graphlayer.qrc">
          <normaloff>:/graphlayerdeletenodes.png</normaloff>:/graphlayerdeletenodes.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeEdgeButton">
        <property name="icon">
         <iconset resource="../../../../../../../../../Downloads/QgsGraphLayerPlugin/graphlayer.qrc">
          <normaloff>:/graphlayerdeleteedges.png</normaloff>:/graphlayerdeleteedges.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Add from vector layers</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="topMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="QPushButton" name="addNodeFromLayerButton">
        <property name="icon">
         <iconset resource="../../../../../../../../../Downloads/QgsGraphLayerPlugin/graphlayer.qrc">
          <normaloff>:/graphlayeraddnodesfromlayer.png</normaloff>:/graphlayeraddnodesfromlayer.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addEdgesFromLayerButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../../../Downloads/QgsGraphLayerPlugin/graphlayer.qrc">
          <normaloff>:/graphlayeraddedgesfromlayer.png</normaloff>:/graphlayeraddedgesfromlayer.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../../../../Downloads/QgsGraphLayerPlugin/graphlayer.qrc"/>
 </resources>
 <connections/>
</ui>
